recipes/metamorpheus/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eox pipefail
+
+PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
+METAMORPHEUS_ROOT=$DOTNET_ROOT/tools/metamorpheus
+
+mkdir -p $METAMORPHEUS_ROOT
+cp -r $SRC_DIR/* $METAMORPHEUS_ROOT
+
+if [[ "${build_platform}" == "win-64" ]]; then
+    mv $METAMORPHEUS_ROOT/CMD.exe $METAMORPHEUS_ROOT/metamorpheus.exe
```

Comment:
```suggestion
    mv $METAMORPHEUS_ROOT/CMD.exe $METAMORPHEUS_ROOT/metamorpheus.exe
    cp $RECIPE_DIR/metamorpheus.cmd $DOTNET_ROOT/tools/
```
**`metamorpheus.cmd:`**
```batch
@"%~dp0\metamorpheus\metamorpheus.exe" %*
```